packages/NodeTypeResolver/src/PerNodeTypeResolver/MethodCallTypeResolver.php

```diff
@@ -68,7 +69,12 @@ public function getNodeClasses(): array
     public function resolve(Node $methodCallNode): array
     {
         $parentCallerTypes = $this->resolveMethodCallVarTypes($methodCallNode);
-        $methodName = (string) $methodCallNode->name;
+
+        if ($methodCallNode->name instanceof Identifier) {
+            $methodName = $methodCallNode->name->toString();
+        } else {
+            $methodName = (string) $methodCallNode->name;
```

Comment:
I wonder, does it make sense to rename `__CLASS__`?

I think we should return empty array, if not `Identifier`:

```php
if (! $methodCallNode->name instanceof Identifier) {
    return [];
}

$methodName = $methodCallNode->name->toString();
```

Reply:
Yeah, makes sense. I'll change the logic :man_technologist: